ui/pages/Project/components/RnaSeqResultPage.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Loader, Grid } from 'semantic-ui-react'
+
+const RnaSeqOutliers = React.lazy(() => import('./RnaSeqOutliers'))
+const RnaSeqSpliceOutliers = React.lazy(() => import('./RnaSeqSpliceOutliers'))
+
+const RnaSeqResultPage = ({ match }) => (
+  <Grid>
+    {match.params.hasRnaOutlierData === 'true' && (
+      <Grid.Row>
+        <Grid.Column width={6}>
+          <React.Suspense fallback={<Loader />}>
+            {React.createElement(RnaSeqOutliers,
+              { familyGuid: match.params.familyGuid, individualGuid: match.params.individualGuid }) }
+          </React.Suspense>
+        </Grid.Column>
+      </Grid.Row>
+    )}
+    {match.params.hasRnaSpliceOutlierData === 'true' && (
+      <Grid.Row>
+        <React.Suspense fallback={<Loader />}>
+          {React.createElement(RnaSeqSpliceOutliers,
+            { familyGuid: match.params.familyGuid, individualGuid: match.params.individualGuid }) }
+        </React.Suspense>
+      </Grid.Row>
+    )}
+  </Grid>
```

Comment:
Still open to how to layout the plots.

Reply:
So at a high level, you want the data loader to be in this page, and do a single fetch for all the data for this page. Then you want a generic `RnaSeqOutlierPlot` class which has the link and graph (so basically the existing `RnaSeqOutliers` content) and a new `RnaSpliceJunctionTable` class to display the junction table described in the ticket

I think that could look something like 
```
<DataLoader content={rnaSeqData} contentId={match.params.individualGuid} load={load} loading={loading}>
  <Grid divided>
    <Grid.Row columns={(rnaSeqData?.expressionOutliers && rnaSeqData?.junctionOutliers) ? 2 : 1}> 
      <React.Suspense fallback={<Loader />}>
        {rnaSeqData?.expressionOutliers&& 
          <Grid.Column><RnaSeqOutlierPlot rnaSeqData={rnaSeqData.expressionOutliers} /></Grid.Column>}
        {rnaSeqData?.junctionOutliers && 
          <Grid.Column><RnaSeqOutlierPlot rnaSeqData={rnaSeqData.junctionOutliers} /></Grid.Column>}
      </React.Suspense>
    </Grid.Row>
    {rnaSeqData?.junctionOutliers &&  
      <Grid.Row  columns={1}><Grid.Column><RnaSpliceJunctionTable /></Grid.Column> </Grid.Row>}
  </Grid>
</DataLoader>
```